--- kyotocabinet-1.2.76/configure	2012-05-24 13:31:45.000000000 +0200
+++ kyotocabinet-1.2.76/configure	2018-08-17 10:50:31.555686070 +0200
@@ -3975,40 +3975,6 @@
  presetting ac_cv_c_bigendian=no (or yes) will help" "$LINENO" 5 ;;
  esac
 
-
-# System-depending optimization
-printf 'checking for 64-bit availability... '
-if printf 'main() {}' | $CC -xc -m64 -o config.tmp - >config.tmp 2>&1
-then
-  MYCFLAGS="-m64 $MYCFLAGS"
-  MYCXXFLAGS="-m64 $MYCXXFLAGS"
-  printf 'yes\n'
-else
-  printf 'no\n'
-fi
-if test "$enable_opt" != "no"
-then
-  printf 'checking for CPU optimization availability... '
-  if printf 'main() {}' | $CC -xc -march=native -o config.tmp - >config.tmp 2>&1
-  then
-    MYCFLAGS="-march=native $MYCFLAGS"
-    MYCXXFLAGS="-march=native $MYCXXFLAGS"
-    printf 'yes\n'
-  else
-    printf 'no\n'
-  fi
-fi
-printf 'checking for useless warnings... '
-if printf 'main() {}' | $CC -xc \
-  -Wno-unused-but-set-variable -Wno-unused-but-set-parameter -o config.tmp - >config.tmp 2>&1
-then
-  MYCFLAGS="$MYCFLAGS -Wno-unused-but-set-variable -Wno-unused-but-set-parameter"
-  MYCXXFLAGS="$MYCXXFLAGS -Wno-unused-but-set-variable -Wno-unused-but-set-parameter"
-  printf 'yes\n'
-else
-  printf 'no\n'
-fi
-
 # Atomic operations
 if test "$enable_atomic" != "no"
 then
